code/modules/antagonists/devil/devil.dm

```diff
@@ -89,8 +89,6 @@ GLOBAL_LIST_INIT(devil_suffix, list(" the Red", " the Soulless", " the Master",
 	roundend_category = "devils"
 	antagpanel_category = "Devil"
 	job_rank = ROLE_DEVIL
-	//Don't delete upon mind destruction, otherwise soul re-selling will break.
```

Comment:
well... does this break soul re-selling?

Reply:
According to original PR
> If this would ever fire it would be due to a bug, minds are not deleted outside of roundstart cleanup. Only leads to confusion.

this seems correctly, minds don't delete so